man/scrot.txt

```diff
@@ -2,128 +2,135 @@ NAME
 
   Example:
 
-    $ scrot '%Y-%m-%d_$wx$h.png' -e 'mv $f ~/shots/'
+    $ scrot '%Y-%m-%d_$wx$h.png' -e 'optipng $f'
 
-  This would create a file called something like 2000-10-30_2560x1024.png
-  and move it to your shots directory.
+  This would create a PNG file with a name similar to 2000-10-30_2560x1024.png
+  and optimize it with optipng(1).
 
 SELECTION STYLE
-  When using --select you can indicate the style of the line with --line.
-
-  The following specifiers are recognised:
-
-    style=(solid,dash),width=(range 1 to 8),color="value",
-    opacity=(range 10 to 100),mode=(edge,classic)
-
-  The default style is:
+  When using -s, you can indicate the style of the line with -l.
+
+  -l takes a comma-separated list of specifiers as argument:
+
+    style=STYLE     STYLE is either "solid" or "dash" without quotes.
+    width=NUM       NUM is a pixel count between 1 and 8 inclusive.
+    color="COLOR"   Color is a hexadecimal HTML color code or the name of
+                    a color. HTML color codes are composed of a pound
+                    sign '#' followed by a sequence of 3 2-digit
+                    hexadecimal numbers which represent red, green, and
+                    blue respectively. Examples: #FF0000 (red), #E0FFFF
+                    (light cyan), #000000 (black).
+    opacity=NUM     NUM is a percentage between 10 and 100 inclusive. 100
+                    means 100% opaque, 10 means 10% opaque. This is only
+                    effective if a Composite Manager is running.
+    mode=MODE       MODE is either "edge" or "classic" without quotes.
+                    edge is the new selection, classic uses the old one.
+                    "edge" ignores the style specifier and the -f flag,
+                    "classic" ignores the opacity specifier.
+
+  Without the -l option, a default style is used:
 
     mode=classic,style=solid,width=1,opacity=100
 
-  Mode 'edge' ignore: style, --freeze
-
-  Mode 'classic' ignore: opacity
-
-  The 'opacity' specifier is only effective if a Composite Manager is running.
-
-  For the color you can use a name or a hexadecimal value.
-
-    color="red" or color="#ff0000"
-
   Example:
 
-    $ scrot --line style=dash,width=3,color="red" --select
+    $ scrot -l style=dash,width=3,color="red" -s
 
 NOTE FORMAT
-  The following specifiers are recognised for the option --note:
+  The -n option's argument is more arguments:
 
-    -f 'FontName/size'
-    -t 'text'
-    -x position (optional)
-    -y position (optional)
-    -c color(RGBA) (optional)
-    -a angle (optional)
+    -f  'FontName/size'
+    -t  'text'
+    -x  position (optional)
+    -y  position (optional)
+    -c  color(RGBA) (optional)
+    -a  angle (optional)
 
   Example:
 
-    $ scrot --note "-f '/usr/share/fonts/TTF/DroidSans-Bold/40' -x 10 -y 20 -c 255,0,0,255 -t 'Hi'"
+    $ scrot --note "-f '/usr/share/fonts/TTF/DroidSans-Bold/40' -x 10 \\
+    >   -y 20 -c 255,0,0,255 -t 'Hi'"
```

Comment:
Do you agree to remove the `>` character?
The option `--note` should be indicated in short form `-n` as all the previous examples do.